src/components/PeoplePage.tsx

```diff
@@ -1,31 +1,247 @@
+import React, {
+  useMemo, useState, useEffect,
+} from 'react';
 import { PeopleFilters } from './PeopleFilters';
 import { Loader } from './Loader';
 import { PeopleTable } from './PeopleTable';
+import { NewPerson } from '../types';
+import { FilterType, SortType, PropName } from '../types/enum';
+
+interface Props {
+  people: NewPerson[],
+  loading: boolean,
+  sexFilter: FilterType,
+  sexFilterHandler: (value: FilterType) => void,
+  setQuery: (value: string) => void,
+  query: string,
+  deleteQuery: () => void,
+  isError: boolean,
+  fetchPeople: () => Promise<void>;
+}
+
+export const PeoplePage: React.FC<Props> = ({
+  people,
+  loading,
+  sexFilter,
+  sexFilterHandler,
+  setQuery,
+  query,
+  deleteQuery,
+  isError,
+  fetchPeople,
+}) => {
+  const [sortField, setsortField] = useState<string | null>(null);
+  const [sortOrder, setSortOrder] = useState(SortType.original);
+  const [clickCount, setClickCount] = useState(0);
+  const [selectedCentury, setSelectedCentury]
```

Comment:
Would be better to rename this variable to show that it's an array but not the one century.